com.unity.template-hd/Packages/com.unity.template.hd/package.json

```diff
@@ -7,9 +7,47 @@
   "unity": "2022.1",
   "description": "This template utilizes the High Definition Render Pipeline. Making it a good starting point for people focused on high-end graphics that want to develop games for platforms that support Shader Model 5.0 (DX11 and above). \n\nBeyond being tuned for High End visuals this project includes\n\u25aa Shadergraph \n\u25aa Visual Effect Graph  \n\u25aa Presets \n\u25aa Example content",
   "dependencies": {
-    "com.unity.cinemachine": "2.8.0",
+    "com.unity.collab-proxy": "1.9.0",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.textmeshpro": "3.0.6",
+	"com.unity.cinemachine": "2.8.0",
+    "com.unity.feature.development": "1.0.1",
     "com.unity.inputsystem": "1.1.0-pre.5",
     "com.unity.learn.iet-framework": "2.0.0",
+    "com.unity.learn.iet-framework.authoring": "1.0.0",
     "com.unity.render-pipelines.high-definition": "13.0.0"
```

Comment:
You're missing a coma `,` at the end of the line, Yamato is broken because of it